aml_module/conda.yaml

```diff
@@ -0,0 +1,14 @@
+name: project_environment
+channels:
+  - defaults
+dependencies:
+  - python=3.6.8
+  - cython=0.29.2
+  - numpy=1.18.1
+  - pip:
+    - pandas==0.25.3
+    - fire==0.3.0
+    - azureml-defaults
+    - azureml-designer-core==0.0.31
+    - azureml-designer-internal==0.0.20
+    - git+https://github.com/microsoft/anomalydetector.git
```

Comment:
You could use additionalIncludes to include the code in the repo instead of referencing the repo in pip.
See https://github.com/microsoft/recommenders/blob/master/reco_utils/azureml/azureml_designer_modules/module_specs/sar_train.yaml#L47